core/src/main/java/org/apache/accumulo/core/client/ScannerBase.java

```diff
@@ -242,4 +242,28 @@
    * @since 1.8.0
    */
   long getBatchTimeout(TimeUnit timeUnit);
+
+  /**
+   * Sets the name of the classloader context on this scanner
+   *
+   * @param context
+   *          name of the classloader context
+   * @since 1.8.0
+   */
+  void setContext(String context);
```

Comment:
Might be better to use the full "classloadContext". "Context" on its own is kind of obtuse and could overlap with other things in the future.